specs/simple-serialize.md

```diff
@@ -79,7 +79,15 @@ For convenience we alias:
 
 ### Default values
 
-The default value of a type upon initialization is recursively defined using `0` for `uintN`, `False` for `boolean` and the elements of `Bitvector`, and `[]` for lists and `Bitlist`. Unions default to the first type in the union (with type index zero), which is `null` if present in the union.
+| Type | Default Value
```

Comment:
A definition for `Container` is missing.

Reply:
Good catch, I just added it!